src/scripts/main.js

```diff
@@ -1,3 +1,64 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+
+const appendRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const appendColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+const max = 10;
+const min = 2;
+
+appendRowButton.addEventListener('click', function() {
+  if (tbody.children.length < max) {
+    removeRowButton.disabled = false;
+    tbody.append(tbody.lastElementChild.cloneNode(true));
+  }
+
+  if (tbody.children.length === max) {
```

Comment:
it's possible to change `if (...)` to just `else`, bc you have similar condition before